### PR TITLE
i18l: Hebrew translation updates

### PR DESCRIPTION
**Pull request by seefood**

### CODE REVIEW

File: config/locales/he.yml [hunk 1]
```diff
@@ -257,7 +257,7 @@ he:
       muting: רשימת השתקות
     upload: יבוא
   landing_strip_html: <strong>%{name}</strong> על השרת <strong>%{domain}</strong>. ניתן לעקוב אחריהם או לדבר איתם בעזרת חשבון על כל שרת קהילה ברחבי הפדרציה. ליצירת חשבון, ניתן<a href="%{sign_up_path}">להרשם כאן</a>.
-  landing_strip_signup_html: If you don't, you can <a href="%{sign_up_path}">sign up here</a>.
+  landing_strip_signup_html: אחרת, ניתן <a href="%{sign_up_path}">להרשם כאן</a>.
```
Comment (Gargron) [inline config/locales/he.yml]: Remove this from the one above

**Review by Gargron — CHANGES_REQUESTED** [review]

File: config/locales/he.yml (hunk 1 above)
Comment (seefood) [inline config/locales/he.yml]: Thanks for noticing. I use meld to pop out changed keys, I haven't noticed the change in value.

went over again, found a few more changes, pushed a fix.

**Review by Gargron — APPROVED** [review]

File: config/locales/simple_form.he.yml [hunk 2]
```diff
@@ -5,9 +5,13 @@ he:
       defaults:
         avatar: 'PNG, GIF או JPG. מקסימום 2MB. גודל התמונה יוקטן ל-120x120px'
         display_name: 'מקסימום 30 תווים'
```
Comment (Gargron) [inline config/locales/simple_form.he.yml]: Syntax error, can't be value + children at the same time, either one or the other

**Review by Gargron — CHANGES_REQUESTED** [review]

File: config/locales/simple_form.he.yml (hunk 2 above)
Comment (seefood) [inline config/locales/simple_form.he.yml]: True.... one of those keys that became a node. pushing a fix.

**Review by Gargron — APPROVED** [review]

**Pull request merged**